homeassistant/components/modbus/switch.py

```diff
@@ -76,7 +75,7 @@
 )
 
 
-async def async_setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, add_entities, _):
```

Comment:
I prefer the old version, where we set a. default value for discovery_info, other integrations use that, and it seems to be standard.

Reply:
OK, I will put it back.